Ginger/GingerCoreCommon/UIElement/ElementInfo.cs

```diff
@@ -218,21 +219,83 @@ public string ElementTypeEnumDescription
             }
         }
 
+        List<String> mOptionalValues = new List<string>();
+        public List<String> OptionalValues
```

Comment:
please add comment using /// to OptionalValues to stop using it and that it is left for backward support only

Reply:
Added comment